e2e/e2etests/test_bitcoin_deposit_and_withdraw_with_dust.go

```diff
@@ -0,0 +1,58 @@
+package e2etests
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/node/e2e/contracts/withdrawer"
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/e2e/utils"
+	crosschaintypes "github.com/zeta-chain/node/x/crosschain/types"
+)
+
+// TestBitcoinDepositAndWithdrawWithDust deposits Bitcoin and call a smart contract that withdraw dust amount
+// It tests the edge case where during a cross-chain call, a invaild withdraw is initiated (processLogs fails)
+func TestBitcoinDepositAndWithdrawWithDust(r *runner.E2ERunner, args []string) {
+	// Given "Live" BTC network
+	stop := r.MineBlocksIfLocalBitcoin()
+	defer stop()
+
+	require.Len(r, args, 0)
+
+	// ARRANGE
+
+	// Deploy the withdrawer contract on ZetaChain with a withdraw amount of 100 satoshis (dust amount is 1000 satoshis)
+	withdrawerAddr, tx, _, err := withdrawer.DeployWithdrawer(r.ZEVMAuth, r.ZEVMClient, big.NewInt(100))
+	require.NoError(r, err)
+
+	// Wait for the transaction to be mined
+	receipt := utils.MustWaitForTxReceipt(r.Ctx, r.ZEVMClient, tx, r.Logger, r.ReceiptTimeout)
+	require.Equal(r, receipt.Status, uint64(1))
+
+	// Given a list of UTXOs
+	utxos, err := r.ListDeployerUTXOs()
+	require.NoError(r, err)
+	require.NotEmpty(r, utxos)
+
+	// ACT
+	// Deposit 0.01 BTC to withdrawer, this is an arbitrary amount, must be greater than dust amount
+	txHash, err := r.SendToTSSFromDeployerWithMemo(0.01, utxos, withdrawerAddr.Bytes())
```

Comment:
This looks to me a pure `deposit` of 0.01 BTC. How would the `onCrossChainCall` on withdrawer contract be called in this scenario? a bit confused

Reply:
Mmh weird, the logs shows smart contract call happens

Reply:
Ok, just checked, independently of the message, we do a contract call if dest is a contract in the V1 workflow

Reply:
okay. That makes sense then.